hack/conformance.sh

```diff
@@ -12,11 +14,25 @@ fi
 export KUBECONFIG=$(cluster-up/kubeconfig.sh)
 
 teardown() {
+    rv=$?
     ./sonobuoy status --json
     ./sonobuoy logs > ${ARTIFACTS}/sonobuoy.log
     results_tarball=$(./sonobuoy retrieve)
     tar -xvzf $results_tarball plugins/e2e/results/
     cp -f $(find plugins/e2e/results/* -name "*.xml") ${ARTIFACTS}/
+
+    if [ $rv -ne 0 ]; then
+        echo "error found, exiting"
+        exit $rv
```

Comment:
Just to understand, Is there special reason to use exit and not return inside teardown() function?

Reply:
exit is more explicit in this context, and we do want to exit with a specific ret code, i dont know by heart if return would do here the same, but anyway wanted the known explicit method